packages/keyring-eth-hd/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add `HdKeyringV2` class implementing `KeyringV2` interface ([#398](https://github.com/MetaMask/accounts/pull/398))
   - Wraps legacy `HdKeyring` to expose accounts via the unified `KeyringV2` API and the `KeyringAccount` type.
 
+### Changed
+
+- `HdKeyringV2` now extends `EthKeyringWrapper` for common Ethereum logic ([#404](https://github.com/MetaMask/accounts/pull/404))
+
```

Comment:
Since it's unreleased, maybe we can group this entry with the `### Added` one directly? like #398 + #404?